src/components/BetaAndLegalNotice.tsx

```diff
@@ -9,14 +9,12 @@ export default function BetaAndLegalNotice(): ReactElement {
       message={
         <>
           <Heading type='2' className='h4 mb-[0.313rem]'>
-            This is a beta for the small business lending data submission
-            platform
+            This is a beta for the Small Business Lending Data Filing Platform
           </Heading>
           <Paragraph>
-            Thank you for participating. The data submission platform is
-            available to upload, test, and validate data. All uploaded data is
-            for testing purposes only and may be removed at any time. For
-            questions or feedback,{' '}
+            Thank you for participating. The platform is available to upload,
```

Comment:
@billhimmelsbach 
Add "beta" so change to: 
Thank you for participating. The beta platform is available to upload,

Reply:
[Changed](https://github.com/cfpb/sbl-frontend/pull/883/commits/c24c7abd4d185c0b2efb0cadbc88e595ac766914)!